test/test_utils.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include <string>
+
+/**
+ * Run an external command given by cmd, and return the results outputted to STDOUT as a string.
+ */
+std::string exec(const char* cmd);
```

Comment:
Please rename this function to something like `execmd` or `execomm` because the name `exec` kind of conflicts with the system call (and also a linux command) with the same name to avoid ambiguity.

Reply:
Good point! I am on it.